source/GstreamerCatLog.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 Sky UK
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation;
+ * version 2.1 of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+#ifndef GSTREAMER_CAT_LOG_H
+#define GSTREAMER_CAT_LOG_H
+
+#include <gst/gst.h>
+
+GST_DEBUG_CATEGORY_EXTERN(rialtoGStreamerCat);
+#define GST_CAT_DEFAULT rialtoGStreamerCat
```

Comment:
I'm not sure if defining GST_CAT_DEFAULT in the header file is safe. It can pollute different headers.
Meaning that if you would have cpp file with below includes:
#include "GstreamerCatLog.h"
#include "differentHeader.h"

then GST_CAT_DEFAULT would be accessible in differentHeader.h in that pareticular translation unit and that might lead to unpredictable behaviour :/